<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MultiPartImageSequenceWidget</class>
 <widget class="QWidget" name="MultiPartImageSequenceWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>379</width>
    <height>466</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <item row="0" column="0">
    <widget class="QLabel" name="directoryLabel">
     <property name="text">
      <string>Directory</string>
     </property>
    </widget>
   </item>
   <item row="3" column="0">
    <widget class="QLabel" name="datasetNameLabel">
     <property name="text">
      <string>Data set name</string>
     </property>
    </widget>
   </item>
   <item row="2" column="0">
    <widget class="QLabel" name="imagesLabel">
     <property name="text">
      <string>Image(s)</string>
     </property>
    </widget>
   </item>
   <item row="5" column="0" colspan="4">
    <widget class="QGroupBox" name="subsamplingGroupBox">
     <property name="minimumSize">
      <size>
       <width>0</width>
       <height>0</height>
      </size>
     </property>
     <property name="toolTip">
      <string>Subsampling configuration</string>
     </property>
     <property name="title">
      <string>Subsampling</string>
     </property>
     <layout class="QGridLayout" name="gridLayout_3">
      <item row="0" column="0">
       <layout class="QGridLayout" name="subsamplingGridLayout">
        <item row="1" column="0">
         <widget class="QLabel" name="resamplingFilterLabel">
          <property name="text">
           <string>Resampling filter</string>
          </property>
         </widget>
        </item>
        <item row="0" column="0">
         <widget class="QLabel" name="subsamplingRatioLabel">
          <property name="text">
           <string>Ratio (%)</string>
          </property>
         </widget>
        </item>
        <item row="1" column="1" colspan="2">
         <widget class="QComboBox" name="resamplingFilterComboBox">
          <property name="toolTip">
           <string>The type of resampling filter to use</string>
          </property>
          <item>
           <property name="text">
            <string>Box</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>Bilinear</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>B-spline</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>Bicubic</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>Carmull-rom</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>Lanczos</string>
           </property>
          </item>
         </widget>
        </item>
        <item row="0" column="1" colspan="2">
         <widget class="QDoubleSpinBox" name="subsamplingRatioSpinBox">
          <property name="toolTip">
           <string>Subsampling ratio </string>
          </property>
          <property name="minimum">
           <double>10.000000000000000</double>
          </property>
          <property name="maximum">
           <double>100.000000000000000</double>
          </property>
          <property name="singleStep">
           <double>10.000000000000000</double>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
   <item row="3" column="1" colspan="3">
    <widget class="QLineEdit" name="datasetNameLineEdit">
     <property name="toolTip">
      <string>The name of the data set in the HDPS GUI</string>
     </property>
    </widget>
   </item>
   <item row="2" column="1" colspan="3">
    <widget class="QListWidget" name="imagesListWidget">
     <property name="minimumSize">
      <size>
       <width>0</width>
       <height>200</height>
      </size>
     </property>
    </widget>
   </item>
   <item row="0" column="1" colspan="2">
    <widget class="QLineEdit" name="directoryLineEdit">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="sizePolicy">
      <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
    </widget>
   </item>
   <item row="0" column="3">
    <widget class="QPushButton" name="directoryPushButton">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="minimumSize">
      <size>
       <width>40</width>
       <height>0</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>40</width>
       <height>16777215</height>
      </size>
     </property>
     <property name="toolTip">
      <string>Click to pick directory containing image sequence</string>
     </property>
     <property name="text">
      <string>...</string>
     </property>
    </widget>
   </item>
   <item row="6" column="0" colspan="4">
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QLineEdit" name="infoLineEdit">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="toolTip">
        <string>Info</string>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="loadSequencePushButton">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="toolTip">
        <string>Load image sequence</string>
       </property>
       <property name="text">
        <string>Load</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
